module/purpose/Defender/src/Defender.hpp

```diff
@@ -43,6 +43,10 @@ namespace module::purpose {
 
         /// @brief Stores configuration values
         struct Config {
+            /// @brief Ready position to walk to (x, y, theta)
+            Eigen::Vector3d ready_position = Eigen::Vector3d::Zero();
+            /// @brief Penalty defence position to walk to when defending from a penalty kick from the opposing team
+            Eigen::Vector3f penalty_defence_position = Eigen::Vector3f::Zero();
```

Comment:
change this to a double and then you don't need to do all the casting 

```suggestion
            Eigen::Vector3d penalty_defence_position = Eigen::Vector3f::Zero();
```

Reply:
omg thanks girl